src/utils/RunSUSHI.js

```diff
@@ -0,0 +1,46 @@
+import { FSHTank, RawFSH } from 'fsh-sushi/dist/import';
+import { exportFHIR } from 'fsh-sushi/dist/export';
+import { logger, Type } from 'fsh-sushi/dist/utils';
+import { FHIRDefinitions } from 'fsh-sushi/dist/fhirdefs';
+import { loadExternalDependencies, fillTank, readConfig } from './Processing';
+
+export async function runSUSHI(input) {
+  // Hard Code config
+  let config = null;
+  try {
+    config = readConfig();
+  } catch {
+    logger.error('Something went wrong when creating the configuration');
+    return;
+  }
```

Comment:
I think we can actually get away with a more lightweight approach here. Most of the config is actually not necessary for turning FSH -> JSON, which is what the Playground is doing. Most of the config is for the IG Publisher, but the Playground (at least at this point) does not care about the IG Publisher. The only property that should really be strictly necessary is `canonical`. So I think you can replace all this code with just:
```
const config = { canonical: "http://default.org" }
```
So `minimalConfig` could go, the `readConfig` function could go, and SUSHI should be able to run export to JSON just fine with only the `canonical`. I'm almost definitely the goof who told you to use `minimalConfig` in the first place, so sorry about that.

Reply:
Sounds good that made things way easier.